pegasus/sites.v3/code.org/views/csf_languages.haml

```diff
@@ -0,0 +1,6 @@
+%h2 
+  %a{href: "/educate/curriculum/elementary-school"} CS Fundamentals (Courses 1-4) is supported in the following languages
```

Comment:
Oh yeah and can you make only the course name hyperlinked? (repeat for all views)


```%h2
  %a{href: "/educate/curriculum/elementary-school"} CS Fundamentals (Courses 1-4)
  is supported in the following languages```